hack/generator/pkg/astmodel/resource.go

```diff
@@ -92,7 +94,7 @@ func (definition *ResourceDefinition) AsDeclarations(codeGenerationContext *Code
 
 	packageName, err := codeGenerationContext.GetImportedPackageName(MetaV1PackageReference)
 	if err != nil {
-		panic(fmt.Errorf("resource definition for %s failed to import package: %w", definition.typeName, err))
+		panic(errors.Wrapf(err, "resource definition for %s failed to import package", definition.typeName))
```

Comment:
This error message seems to be an incomplete sentence, perhaps rephrase?

Reply:
I think it reads alright with the error on the end? So maybe `resource definition for ThisType failed to import package: error reading file "blah.json"`?